examples/ensemble/plot_adaboost_hastie_10_2.py

```diff
@@ -50,6 +60,14 @@
 dt.fit(X_train, y_train)
 dt_err = 1.0 - dt.score(X_test, y_test)
 
+# %%
+# Adaboost with discrete SAMME and real SAMME.R
+# ---------------------------------------------
+# We now define the discrete and real AdaBoost classifiers
+# and fit them to the training set.
+
+from sklearn.ensemble import AdaBoostClassifier
+
```

Comment:
Can you add a marker `# %%` in l.78 to get a diagram for both models?

Reply:
Good catch. Yes